packages/editor/src/components/editor-history/redo.js

```diff
@@ -24,11 +27,15 @@ function EditorHistoryRedo( { hasRedo, redo } ) {
 	);
 }
 
-export default compose( [
+const EnhancedEditorHistoryRedo = compose( [
```

Comment:
Have you considered refactoring this and other similar components (`EditorHistoryUndo` or `TableOfContents`) to use hooks? I think it would remove the need to use `innerRef` and you could apply `ref` directly on the component.

Reply:
Sure thing👍